docker/Dockerfile.requirements

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1.4
-ARG PYTHON_IMAGE=python:3.12.2-slim-bookworm
+ARG PYTHON_IMAGE=python:3.12-slim-bookworm
```

Comment:
I think we should pin the image to `3.12.4` because I think if they were to release a `3.12.5` this would automatically move over to that

Reply:
Do u think we should change API as well?